Makefile

```diff
@@ -13,7 +13,7 @@ include cheshire.mk
 
 # Inside the repo, forward (prefixed) all, nonfree, and clean targets
 all:
-	@$(MAKE) chs-all
+	@$(MAKE) chs-all 
```

Comment:
Remove trailing spaces.

Reply:
but actually they are aligned with the rest of the mk file

Reply:
The trailing space is after the `chs-all`